tests/AgDatabaseMove.Unit/SmoFacadeListenerTest.cs

```diff
@@ -0,0 +1,86 @@
+﻿using System.Collections.Generic;
+using Xunit;
+
+namespace AgDatabaseMove.Unit
+{
+  public class SmoFacadeListenerTest
+  {
+
+    /// <summary>
+    /// Tests for Listener.SplitDomainAndPort()
+    /// </summary>
+
+    private const string DOMAIN = "abc.def.ghi";
+    private const string HOST = "abc";
+
+    // {input, expectedDomain, expectedPort (can be null)}
+    public static IEnumerable<object[]> ValidPorts => new List<object[]> {
+      new object[] { $"{DOMAIN}", DOMAIN },
+      new object[] { $"{DOMAIN},123", DOMAIN, ",123" },
+      new object[] { $"{DOMAIN}\\SQL", DOMAIN, "\\SQL" },
+
+      new object[] { $"{HOST}", HOST },
+      new object[] { $"{HOST},123", HOST, ",123" },
+      new object[] { $"{HOST}\\SQL", HOST, "\\SQL" }
+    };
+
+    [Theory]
+    [MemberData(nameof(ValidPorts))]
```

Comment:
`[InlineData]` is a bit more simple if you're only going to use the Enumerable inputs once
https://hamidmosalla.com/2017/02/25/xunit-theory-working-with-inlinedata-memberdata-classdata/

Reply:
Looks like we can't use `[InlineData]` for the first 2 tests - it doesn't support interpolated strings
Will change the last one